auth_app/models.py

```diff
@@ -1 +1,46 @@
-# from django.db import models
+from django.contrib.auth.models import (
+    AbstractBaseUser,
+    BaseUserManager,
+    PermissionsMixin)
+from django.db import models
+from django.utils import timezone
+
+
+class UserManager(BaseUserManager):
+    def create_user(self, email, username, password):
+        if not email:
+            raise ValueError("ENTER AN EMAIL BUDDY")
+        if not username:
+            raise ValueError("I KNOW YOU HAVE A NAME")
+        if not password:
+            raise ValueError("PASSWORD?!?!?!? HELLO??")
```

Comment:
Is this necessary? Django raises a `IntegrityError` when trying to store a `NULL` value in the DB